types/tektelic/edoctorV1/default.schema.json

```
@@ -0,0 +1,62 @@
+{
+  "$id": "https://akenza.io/tektelic/edoctorV1/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "topic": "default",
+  "processingType": "uplink_decoder",
+  "properties": {
+    "skinTemperature": {
+      "title": "Skin temperature",
+      "unit": "°C",
+      "type": "number",
+      "description": "Skin temperature"
+    },
+    "respiratoryRate": {
+      "title": "Respiratory rate",
+      "unit": "bpm",
+      "type": "number",
+      "description": "Respiratory rate"
+    },
+    "uaModeActive": {
+      "title": "UA mode active",
+      "type": "boolean",
+      "description": "UA mode active"
+    },
+    "chestExpansion": {
```

We could add a `minimum` and `maximum` here (as this is always known when calculating a percentage).

```suggestion
    "chestExpansion": {
      "title": "Chest expansion",
      "unit": "%",
      "type": "number",
      "minimum": 0.0,
      "maximum": 100.0,
      "description": "Chest expansion"
    },
```